internal/featuresupport/features_olm_operators_test.go

```diff
@@ -236,4 +236,85 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 		Entry("on S390x is    NOT supported", []string{"4.11", "4.13", "4.14", "4.21"}, models.ClusterCPUArchitectureS390x, false),
 		Entry("on ppc64le is	NOT supported", []string{"4.11", "4.13", "4.14", "4.21"}, models.ClusterCPUArchitecturePpc64le, false),
 	)
+
+	Context("Test MTV feature", func() {
+		It("Validate MTV on CPU arch", func() {
+			supportedCpuArch := []string{
+				models.ClusterCPUArchitectureArm64,
+				models.ClusterCPUArchitectureMulti,
+				models.ClusterCPUArchitectureX8664,
+			}
+			notSupportedCpuArch := []string{
+				models.ClusterCPUArchitectureS390x,
+				models.ClusterCPUArchitecturePpc64le,
+			}
+			for _, arch := range supportedCpuArch {
+				Expect(IsFeatureAvailable(models.FeatureSupportLevelIDMTV, "4.14", swag.String(arch))).To(BeTrue())
+			}
+			for _, arch := range notSupportedCpuArch {
+				Expect(IsFeatureAvailable(models.FeatureSupportLevelIDMTV, "4.14", swag.String(arch))).To(BeFalse())
+			}
+		})
+		It("Validate Feature Support for MTV", func() {
+
+			tests := []struct {
+				id       int // used to know which test case failed
+				version  string
+				platform *models.PlatformType
+				expected models.SupportLevel
+			}{
+				{
+					id:       1,
+					version:  "4.11",
+					platform: models.PlatformTypeNone.Pointer(),
+					expected: models.SupportLevelUnavailable,
+				},
+				{
+					id:       2,
+					version:  "4.9",
+					platform: models.PlatformTypeBaremetal.Pointer(),
+					expected: models.SupportLevelUnavailable,
+				},
+				{
+					id:       3,
+					version:  "4.11",
+					platform: models.PlatformTypeVsphere.Pointer(),
+					expected: models.SupportLevelUnavailable,
+				},
+				{
+					id:       4,
+					version:  "4.12",
+					platform: models.PlatformTypeBaremetal.Pointer(),
+					expected: models.SupportLevelUnavailable,
+				},
+				{
+					id:       5,
+					version:  "4.14",
+					platform: models.PlatformTypeNone.Pointer(),
+					expected: models.SupportLevelSupported,
+				},
+				{
+					id:       6,
+					version:  "4.15",
+					platform: models.PlatformTypeNone.Pointer(),
+					expected: models.SupportLevelSupported,
+				},
+			}
+
+			for _, test := range tests {
+				featureSupportParams := SupportLevelFilters{OpenshiftVersion: test.version, CPUArchitecture: nil, PlatformType: test.platform}
+				resultSupportLevel := GetSupportLevel(models.FeatureSupportLevelIDMTV, featureSupportParams)
+				Expect(fmt.Sprintf("id: %d, result: %s", test.id, resultSupportLevel)).To(Equal(fmt.Sprintf("id: %d, result: %s", test.id, test.expected)))
+			}
```

Comment:
just FYI, instead of using `id` for debuging you can send a string which will output on failure
```go
Expect(fmt.Sprintf(resultSupportLevel)).To(Equal(test.expected),
     fmt.Sprintf("failed %v,  with result %v", id, resultSupportLevel))
```